tests/tests_geomstats/test_backends.py

```diff
@@ -790,7 +789,7 @@ def test_choice(self):
             result_bool = False
 
         self.assertTrue(result_bool)
-        self.assertEqual(len(result), a)
+        assert len(result) == a
```

Comment:
NIT: why not self.assertEqual(len(result), a)?

Reply:
yup.. first I thought why equal...  Then I was experimenting.